content/english/useless-box/build-the-box.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Build the box"
+description: ""
+draft: true
+weight: 3
+---
+
+## Let's build the box
+![diagram of the parts to build the box, base, lids, sides, end, motor mount and finger](../img/boxPartsDiagram.png)
+1.	Glue 2 fingers together, to make a thicker finger. (If it�s already thick enough, skip this step)
```

Comment:
How thick is "thick enough"? Can you give an exact measurement in cm/inches?

```suggestion
1.	Glue 2 fingers together, to make a thicker finger. (If it's already thick enough, skip this step).
```

Reply:
this will get updated, as we're taking pictures and will provide more details on the materials